package.xml

```diff
@@ -4,10 +4,11 @@
   <version>0.0.1</version>
   <description>A Human hand model as URDF</description>
 
-  <maintainer email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</maintainer>
-  <license>NO REDISTRIBUTION ACCEPTED</license>
+  <maintainer email="rhaschke@techfak.uni-bielefeld.de">Robert Haschke</maintainer>
+  <license>GPLv2</license>
   
-  <author>Guillaume Walck</author>
+  <author email="rhaschke@techfak.uni-bielefeld.de">Robert Haschke</author>
+  <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
   <author>Carlos Rosales</author>
```

Comment:
Who is this? What did he contributed? Shouldn't this be mentioned somewhere?

Reply:
He is the original author of the package we derived from. See https://github.com/WEARHAP/hand-models/blob/master/pisa_hand_description/package.xml

Reply:
Consider https://github.com/robomantic/human_hand/pull/1 then.

Reply:
Apparently (did not look details) GPLv2 does not enforce author attribution as does GPLv3 (see https://opensource.stackexchange.com/questions/8141/gpl-author-attributions) so your suggestion is a good alternative to not have the past author as a copyright holder of the current work but as an attributed author in a separate file as you did in https://github.com/robomantic/human_hand/pull/1